services/one-stream/package/buildAnyPackage.js

```diff
@@ -191,6 +192,22 @@ export const buildAnyPackage = async (packageId, config) => {
       putParams.Item.GSI1sk = packageId;
     } else {
       console.log("%s is not a OneMAC package: ", packageId, putParams.Item);
+      // if there was a package, and should not be now, delete the package item
```

Comment:
Nice! Im not sure what the use case for this is outside of soft delete but it makes sense!

Reply:
I *think* it would also remove the package if the id got changed - I know you delete the package in the lambda you made, but in case someone just goes into the data and edits it, instead of using the lambda...